LayoutTests/crypto/subtle/ed25519-generate-export-key-raw.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="../../resources/js-test-pre.js"></script>
+<script src="../resources/common.js"></script>
+</head>
+<body>
+<p id="description"></p>
+<div id="console"></div>
+
+<script type="text/javascript">
+description("Test exporting an Ed25519 public key with raw format.");
+
+jsTestIsAsync = true;
+
+var algorithmKeyGen = {
+    name: "Ed25519",
+    namedCurve: "Curve25519"
+};
+var extractable = true;
+
+var keyPair;
+debug("Generating a key pair...");
+crypto.subtle.generateKey(algorithmKeyGen, extractable, ["sign", "verify"]).then(function(result) {
```

Comment:
It would be nice if these tests were written as WPT tests (using testharness.js), since they can be used by other browsers.

Reply:
I have undone the changes on the WPT and I think that would be better as a follow-up patch